apps/stress-test/src/pages/wc/simple-stress/index.ts

```diff
@@ -1,11 +1,11 @@
-import {
-  fluentButton,
-  fluentDivider,
-  fluentCheckbox,
-  fluentProgressRing,
-  fluentNumberField,
-  provideFluentDesignSystem,
-} from '@fluentui/web-components';
+// import {
```

Comment:
Rebasing from `master` will remove the stress-test app and the need for these changes.

Reply:
Tracking - thank you!